obj[]   sameObj # lista oggetti
float32 correlation    # correlation value